requirements.txt

```diff
@@ -1,2 +1 @@
 Cython==0.24.1
-numpy==1.11.1
```

Comment:
Running `python setup.py install` automatically installs `numpy`, as you can also see from the test and the `.travis.yml` file. I noticed that keeping `numpy` in the `requirements.txt` was not a necessity.